### PR TITLE
Add option to get nodes without using event-counts

### PR DESCRIPTION
**Pull request by gdubicki**
As event-counts query may be very slow and has experimental
status it is beneficial for some users to get nodes without
exact number of changed resources in the last report
for compatibility and performance benefit.

### CODE REVIEW

**Comment by gdubicki** [comment]
See https://github.com/voxpupuli/puppetboard/issues/430 and https://github.com/voxpupuli/puppetboard/issues/149 .

**Comment by daenney** [comment]
This seems reasonable but you broke the line length requirement now. Could you fix that please?

**Comment by gdubicki** [comment]
> Could you fix that please?

Fixed.

File: pypuppetdb/api.py [hunk 1]
```diff
@@ -424,25 +433,39 @@ def nodes(self, unreported=2, with_status=False, **kwargs):
             node['events'] = None
 
             if with_status:
-                status = [s for s in latest_events
-                          if s['subject']['title'] == node['certname']]
+                if with_event_numbers:
+                    status = [s for s in latest_events
```
Comment (othalla) [inline pypuppetdb/api.py]: as status is defined inside if statement so it may be not defined later

**Comment by bastelfreak** [comment]
Thanks @gdubicki !

**Pull request merged**